sdk/helper/ldaputil/config.go

```diff
@@ -105,6 +114,19 @@ Default: cn`,
 		"certificate": {
 			Type:        framework.TypeString,
 			Description: "CA certificate to use when verifying LDAP server certificate, must be x509 PEM encoded (optional)",
+			DisplayAttrs: &framework.DisplayAttributes{
+				Name: "CA certificate",
```

Comment:
since this is a certificate, it needs the `EditType: "file",` beneath the name here. this ensures a certificate file can be uploaded in the UI.

Reply:
Thank you noelledaley! This is the piece I was missing last time I tried this in order to get the nice file uploader in the UI.

Reply:
@Gisson would be great if you could address @noelledaley comment here. Thanks!

Reply:
@michelvocks fixed.